addons/report/tests/test_reports.py

```diff
@@ -2,35 +2,34 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 import logging
 
-import openerp
-import openerp.tests
+import odoo
+import odoo.tests
 
 
 _logger = logging.getLogger(__name__)
 
 
-@openerp.tests.common.at_install(False)
-@openerp.tests.common.post_install(True)
-class TestReports(openerp.tests.TransactionCase):
+@odoo.tests.common.at_install(False)
+@odoo.tests.common.post_install(True)
+class TestReports(odoo.tests.TransactionCase):
     def test_reports(self):
-        registry, cr, uid = self.registry, self.cr, self.uid
-        r_model = registry('ir.actions.report.xml')
+        r_model = self.env['ir.actions.report.xml']
         domain = [('report_type', 'like', 'qweb')]
-        for r in r_model.browse(cr, uid, r_model.search(cr, uid, domain)):
+        for r in r_model.search(domain):
```

Comment:
If i am wrong then please correct me , as per name convention change the irritable variable name ***r***.
***r*** should be change like : 
```
for report in r_model.search(domain):
```